packages/blast-utilities/config-utils.js

```diff
@@ -3,37 +3,22 @@ const process = require('process')
 const path = require('path')
 const BlastError = require('./blast-error')
 
-let config = {}
-
-const configPath = path.join(process.cwd(), 'blast.config.js')
-const accountsPath = path.join(process.cwd(), 'accounts.json')
-
 function getConfig() {
-  if (!fsExtra.pathExistsSync(configPath)) {
-    throw new BlastError(`Config file was not found! Make sure that blast.config.js exists at ${configPath}`)
+  const CONFIG_PATH = path.join(process.cwd(), 'blast.config.js')
+  if (!fsExtra.pathExistsSync(CONFIG_PATH)) {
+    throw new BlastError(`Config file was not found! Make sure that blast.config.js exists at ${CONFIG_PATH}`)
   }
-  config = require(configPath)
+  config = require(CONFIG_PATH)
```

Comment:
can you define this `config`?

Reply:
This is coming from code styling that is checked by ESLint. There are similar styling errors in `global-functons.js`